WordPress/Classes/ViewRelated/System/Search/WPSearchControllerConfigurator.h

```diff
@@ -0,0 +1,17 @@
+#import <Foundation/Foundation.h>
+#import "WPSearchController.h"
+
+extern const CGFloat SearchBarWidth;
+extern const CGFloat SearchBariPadWidth;
+extern const CGFloat SearchWrapperViewPortraitHeight;
+extern const CGFloat SearchWrapperViewLandscapeHeight;
+extern const NSTimeInterval SearchBarAnimationDuration;
+
+@interface WPSearchControllerConfigurator : NSObject
```

Comment:
Documentation added! However doesn't show up on this code snippet for some reason... interesting